go.mod

```diff
@@ -12,13 +12,15 @@ require (
 	github.com/docker/docker v1.13.1
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
+	github.com/ghodss/yaml v1.0.0
```

Comment:
I don't know much about marshal/unmarshal, but don't we have go-yaml already added ? Is the following better or both have different use-cases?

Reply:
Yes, the purpose of this package is different. This is used to convert YAML obj to JSON obj